libr/util/log.c

```diff
@@ -2,32 +2,34 @@
 
 #include <r_util.h>
 
-// XXX: This api is kinda ugly.. we need to redefine it
+R_API void r_vlog(RLogLevel level, const char *fmt, va_list ap) {
+	const char *hdr = "";
 
-static const char *logfile = "radare.log";
-
-R_API void r_log_file(const char *str) {
-	FILE *fd = r_sandbox_fopen (logfile, "a+");
```

Comment:
you are killing the support for logging into files. WHY

Reply:
because it's not used and we are going to reimplement that in other ways anyway. Not used code == wrong code == dead code == trash code

Reply:
i use it for debugging already, thats why it was there. and it doesnt hurts and its a feature we will want to have when we refactor, and in fact this API looks good to me.

we can probably add a call to r_log_file in radare2.c after r_sys_getenv("R_LOG_FILE") and then it will be used and useful

Reply:
Ok i'll keep it as it is, but I'll also keep my function. My guess is that this function will be removed/changed anyway in the future... again, not used functions are as good as dead :)